### PR TITLE
LWIMIQ - bugs & npm - Oct 29 to Nov 6

### PR DESCRIPTION
**Pull request by himdel**
A new entry for Last Week in ManageIQ.

Apparently the last one went only until the friday 28th, so this one is actually for the 9 days until (including) this Sunday.

### CODE REVIEW

**Comment by JPrause** [comment]
@syncrou @imtayadeway @bronaghs can each of you review and offer up any suggested changes.

File: site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+[Martin](https://github.com/himdel) here, bringing you this week's edition of Last Week in ManageIQ.
+
+
+## Featured
+
+Since the last edition, we've merged 114 [pull requests](https://github.com/ManageIQ/manageiq/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20base%3Amaster%20merged%3A%222016-10-29%20..%202016-11-06%22%20sort%3Acreated-desc%20) in the `manageiq` repo, 56 of which were labeled `bug`, 18 `enhancement`, 16 `technical debt` and 13 `refactoring`. As always, almost half was in the UI, totalling 48 `ui` pull requests.
+
+In the recently renamed `manageiq-ui-service` repo, we've had 8 [pull requests](https://github.com/ManageIQ/manageiq-ui-service/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20base%3Amaster%20merged%3A%222016-10-29%20..%202016-11-06%22%20sort%3Acreated-desc%20), and there's surely more in our other repos.
+
```
Comment (bronaghs) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: LWIMIQ really tests my grammar... should this read "there are" or "there's"?

**Comment by bronaghs** [comment]
@himdel - really great Martin 👍

File: site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md (hunk 1 above)
Comment (himdel) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: I *think* there **are** more pull request, but there **is** more if you don't say what. But obviously I could be wrong :) if you're sure, I'll change it :)

File: site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Wrapping up
+
+Well, that's all folks! See you next week, same place (which BTW got updated since the last time, hope you noticed :)).
+
+
+[open issues]: https://github.com/ManageIQ/manageiq/issues
+[contributors]: https://github.com/ManageIQ/manageiq/graphs/contributors
+[MiQ PRs]: https://github.com/ManageIQ/manageiq/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20base%3Amaster%20merged%3A%222016-10-29%20..%202016-11-06%22%20sort%3Acreated-desc%20
```
Comment (imtayadeway) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: Are these dates running Sat-Sun? i.e. 9 days?

Comment (himdel) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: Yup they are... https://github.com/ManageIQ/manageiq.org/pull/427#issue-187982276 ;)

> Apparently the last one went only until the friday 28th, so this one is actually for the 9 days until (including) this Sunday.

File: site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md (hunk 1 above)
Comment (imtayadeway) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: @himdel I think it depends on what your implied subject is, which does seem to be pull requests here.

Comment (himdel) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: Aah, ok then.. fixing :)

File: site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+In the recently renamed `manageiq-ui-service` repo, we've had 8 [pull requests](https://github.com/ManageIQ/manageiq-ui-service/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20base%3Amaster%20merged%3A%222016-10-29%20..%202016-11-06%22%20sort%3Acreated-desc%20), and there's surely more in our other repos.
+
+There's also many [open issues] you can try your skills on, because who doesn't want to be [famous](https://github.com/ManageIQ/manageiq/graphs/contributors)? And, as always, we're ready to help on [gitter].
+
+
+## Improved
+
+As a sign of things to come, [Allen](https://github.com/AllenBW) has finished moving our Service UI from `bower` to `npm` (actually, `yarn`, we're so modern!) in [#296](https://github.com/ManageIQ/manageiq-ui-service/pull/296), ManageIQ itself will follow shortly.
```
Comment (imtayadeway) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: Tiny nit - but the tools are Bower, npm and Yarn. Their binaries will be `bower`, etc..

File: site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+[Martin](https://github.com/himdel) here, bringing you this week's edition of Last Week in ManageIQ.
+
+
+## Featured
+
+Since the last edition, we've merged 114 [pull requests](https://github.com/ManageIQ/manageiq/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20base%3Amaster%20merged%3A%222016-10-29%20..%202016-11-06%22%20sort%3Acreated-desc%20) in the `manageiq` repo, 56 of which were labeled `bug`, 18 `enhancement`, 16 `technical debt` and 13 `refactoring`. As always, almost half was in the UI, totalling 48 `ui` pull requests.
+
+In the recently renamed `manageiq-ui-service` repo, we've had 8 [pull requests](https://github.com/ManageIQ/manageiq-ui-service/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20base%3Amaster%20merged%3A%222016-10-29%20..%202016-11-06%22%20sort%3Acreated-desc%20), and there's surely more in our other repos.
+
+There's also many [open issues] you can try your skills on, because who doesn't want to be [famous](https://github.com/ManageIQ/manageiq/graphs/contributors)? And, as always, we're ready to help on [gitter].
```
Comment (himdel) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: fixed :)

File: site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md (hunk 3 above)
Comment (himdel) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: aand fixed :) Thanks

**Comment by imtayadeway** [comment]
@himdel great issue! :balloon:

File: site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+## New
+
+[Libor](https://github.com/lpichler) has made it possible to do daily chargeback, in [#12351](https://github.com/ManageIQ/manageiq/pull/12351), following up on his earlier work fixing hourly, weekly and monthly.
+
+From the OpenStack land, [Gilles](https://github.com/gildub) has been adding support for provisioning security groups ([#11758](https://github.com/ManageIQ/manageiq/pull/11758)) and floating IPs ([#11819](https://github.com/ManageIQ/manageiq/pull/11819)), with pull requests to support these in the UI coming up shortly.
+
+In RHEV land, [Tomas](https://github.com/jelkosz) has implemented migrating VMs in [#11366](https://github.com/ManageIQ/manageiq/pull/11366).
+
+And in our API, [Alon](https://github.com/alongoldboim) has added a `MiqPolicy` collection making it possible to CRUD policies ([#11505](https://github.com/ManageIQ/manageiq/pull/11505)), and [Brandon](https://github.com/bdunne) has made it possible to approve or deny requests via the API ([#12258](https://github.com/ManageIQ/manageiq/pull/12258)).
+
```
Comment (syncrou) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: @himdel - I really like the `()` around the PR #'s and was wondering if we should wrap all of them that way? Currently some are and some are not.

**Comment by syncrou** [comment]
@himdel - Great post! - Lots of fun reading here for our audience.  Just the one question about PR #'s.

File: site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md (hunk 5 above)
Comment (himdel) [inline site/_posts/2016-11-07-last-week-in-manageiq-bugs-and-npm.html.md]: yeah, I started doing without, then ran out of ways to say "P has done X in pr N"

But you're right, consistency good, I'll fix it right away :)

**Comment by JPrause** [comment]
@himdel have all final edits been made?

**Comment by himdel** [comment]
@JPrause just a sec  for the final one..

**Comment by himdel** [comment]
@JPrause aand done, @syncrou can you recheck please?

**Comment by syncrou** [comment]
👍 Looks great!

**Comment by JPrause** [comment]
Great,..thanks @himdel @bronaghs @imtayadeway and @syncrou 
Awesome teamwork!

@hayesr please merge away for this week's blog entry.

**Pull request merged**